src/solution/react-dom/react-dom/index.js

```diff
@@ -23,7 +23,7 @@ function render(
 
         if (prevReactElement.type === reactElement.type) {
             prevVNode.update(reactElement);
-            return;
+            return prevVNode.getPublicInstance();
```

Comment:
Hva er grunnen til dette?

Reply:
> ReactDOM.render() currently returns a reference to the root ReactComponent instance. However, using this return value is legacy and should be avoided because future versions of React may render components asynchronously in some cases. If you need a reference to the root ReactComponent instance, the preferred solution is to attach a callback ref to the root element.
https://reactjs.org/docs/react-dom.html#render

Reply:
Det er jo det samme som skjer første gang vi rendrer. Da returnerer vi jo instansen, men så undefined ved et nytt kall med samme komponenten på samme element.